app/src/main/java/com/example/keybindhelper/ui/keybind/KeybindFragment.kt

```diff
@@ -28,67 +28,67 @@ class KeybindFragment : Fragment() {
 
         val root: View = LayoutInflater.from(this.context).inflate(R.layout.fragment_keybind,container,false)//binding.root
 
-        val mainActivity=activity as MainActivity;
-        if(mainActivity.Menu==null)
+        val mainActivity=activity as MainActivity
+        if(mainActivity.menu==null)
             mainActivity.onMenuInit=(object:MainActivity.MenuInitialized{
                 override fun menuHasInitialized() {
-                    initMenu(mainActivity,root);
+                    initMenu(mainActivity,root)
                 }
             })
         else
-            initMenu(mainActivity,root);
+            initMenu(mainActivity,root)
 
         if(CurrentProjectManager.CurrentProject==null || CurrentProjectManager.CurrentProject.Groups==null) {
             CurrentProjectManager.isProjectLoaded.observe(viewLifecycleOwner) {
                 println("LOADED")
-                loadRecycleView(root);
+                loadRecycleView(root)
             }
         }else{
             println("already loaded")
-            loadRecycleView(root);
+            loadRecycleView(root)
         }
 
 
         return root
     }
 
     private fun initMenu(mainActivity: MainActivity,view: View) {
-        mainActivity.setAppBarTitle(CurrentProjectManager.CurrentProject.name.value!!);
+        mainActivity.setAppBarTitle(CurrentProjectManager.CurrentProject.name.value!!)
         mainActivity.showMenuItems(mainActivity.keybindsFragmentActionMenuIds)
-        val rv=view.findViewById<RecyclerView>(R.id.recyclerView);
+        val rv=view.findViewById<RecyclerView>(R.id.recyclerView)
 
-        mainActivity.Menu!!.findItem(R.id.action_delete_all_groups).setOnMenuItemClickListener {
+        mainActivity.menu!!.findItem(R.id.action_delete_all_groups).setOnMenuItemClickListener {
```

Comment:
While you're in here, remove force unwrap (!!).  That is dangerous, as it can lead to NullPointerExceptions.
Consider something null-safe, like menu?.let{} ?: instead.